### ISSUE TITLE
404/500 handlers throwing error

### ISSUE THREAD

**Issue by rbrtmrtn** [CityOfPhiladelphia/ais#26]
Noticed an error in Sentry where [this line] (https://github.com/CityOfPhiladelphia/ais/blob/master/ais/api/views.py#L39) of the 400/500 error handler is throwing an error because the `e` object doesn't have an attribute `code`.

```
@app.errorhandler(404)
@app.errorhandler(500)
def handle_errors(e):
    error = json_error(e.code, e.description, None)  # happening here
    return json_response(response=error, status=e.code)
```

**Comment by rbrtmrtn** [comment]
For an example see https://argus.phila.gov/phila/ais-dev/issues/328/

**Comment by mjumbewu** [comment]
Thanks for the argus link. Should be fixed as of commit e2cd5b7d99b01cfd3d479fd9557651752a310ab2
